src/promptflow/promptflow/_sdk/operations/_experiment_operations.py

```diff
@@ -121,6 +122,24 @@ def stop(self, name: str, **kwargs) -> Experiment:
         ExperimentOrchestrator(self._client, self.get(name)).stop()
         return self.get(name)
 
+    @monitor_operation(activity_name="pf.experiment.run", activity_type=ActivityType.PUBLICAPI)
+    def run(self, experiment: Experiment, stream=False, **kwargs) -> Experiment:
```

Comment:
let's consider the cli experience:

pf experiment start --file xx.exp.yaml
pf experiment start --name xxx

We should also have stop?
pf experiment stop --file xx.exp.yaml
pf experiment stop --name xx?

Or we name this like 
pf experiment stop --experiment xx.exp.yaml/name

Reply:
offline sync:
pf experiment start --name exp_name --> pfclient.experiment.start(experiment=exp_object)
pf experiment start --file path/to/exp.yaml --inputs key=value --stream --> pfclinet.experiment.run(experiment=exp_obj, inputs={}, stream=False)
 
pf experiment stop --name exp_name --> pfclient.experiment.stop(experiment=exp_object)